releases/stellar/2025-01-GMP-v1.0.0.md

```diff
@@ -134,7 +134,7 @@ node stellar/gateway.js rotate
 8. Get the list of operators used by the relayer and register on the Operators contract.
 
 ```bash
-node stellar/operators.js add-operator [operator-address]
+node stellar/operators.js add-operator OPERATOR_ADDRESS
```

Comment:
```suggestion
node stellar/operators.js add-operator
```
Hmm, we've been using `[..]` syntax in the doc unless there's an env var that needs to be set beforehand and used